.gitignore

```diff
@@ -30,3 +30,6 @@ bower_components
 Thumbs.db
 
 dist/css/styles.css
+*.json
+package.json
+package-lock.json
```

Comment:
We need to keep this in the git repository too. See the comment on here https://stackoverflow.com/questions/44206782/do-i-commit-the-package-lock-json-file-created-by-npm-5